src/Compatibility/ControlGallery/src/Issues.Shared/Bugzilla31330.cs

```diff
@@ -107,7 +107,7 @@ public ListItemViewModel(ListViewModel listViewModel)
 			{
 				if (listViewModel == null)
 				{
-					throw new ArgumentNullException("listViewModel");
+					throw new ArgumentNullException(nameof(listViewModel));
```

Comment:
Can we split the PR in 2-3 ones?, covering:
- Compatibility
- Core
- Controls

Would simplify things to test and review.

Reply:
Updated this to just target core for now.